app.R

```diff
@@ -0,0 +1,541 @@
+            tags$li("P(Not T|Not D) is called the Specificity")
+          ),
+          br(),
+          div(style = "text-align: center;",
+              bsButton(
+                inputId = "gochallenge",
+                label = "Go to the Challenge",
+                size = "large",
+                icon = icon("bolt")
+              )
+          )
+        ),
+        #### Set up a Challenge Page ----
+        tabItem(
+          tabName = "challenge",
+          withMathJax(),
+          h2("Challenge Yourself"), 
+          fluidRow(
+            column(
+              width = 6,
+              offset = 0,
+              wellPanel(
+                bsButton(
+                  inputId = "hint",
+                  label = "Show hint", 
+                  icon = icon("question"),
+                  size = "xs"
+                ), 
+                sliderInput(
+                  inputId = "infect", 
+                  label = "Prevalence (Per 1000 People)", 
+                  min = 1, 
+                  max = 100, 
+                  step = 1, 
+                  value = 5),
+                bsPopover(
+                  id = "infect", 
+                  title = "Prevalence (Per 1000 People)",
+                  content = "The average number of people (per 1000) that has the disease.", 
+                  placement = "bottom", 
+                  options = NULL),
+                sliderInput(
+                  inputId = "spec", 
+                  label = "Specificity", 
+                  min = 0.5, 
+                  max = 0.999, 
+                  value = 0.99, 
+                  step = 0.001),
+                bsPopover(
+                  id = "spec", 
+                  title = "Specificity",
+                  content = "The probability of someone who <b>does not have</b> 
+                            the disease testing positive for it.", 
+                  placement = "bottom", options = NULL),
+                sliderInput(
+                  inputId = "sens", 
+                  label = "Sensitivity", 
+                  min = 0.5, 
+                  max = 0.999, 
+                  value = 0.995, 
+                  step = 0.001),
+                bsPopover(
+                  id = "sens", 
+                  title = "Sensitivity",
+                  content = "The probability of someone who <b>has</b> the 
+                            disease testing positive for it.", 
+                  placement = "bottom", 
+                  options = NULL),
+              )
+            ),
+            column(
+              width = 6, 
+              offset = 0,
+              wellPanel(
+                bsButton(
+                  inputId = "ques", 
+                  label = "New Challenge"),
+                br(), 
+                br(),
+                textOutput("question"),
+                br(), 
+                bsButton(
+                  inputId = "show_ans", 
```

Comment:
Might need to add style, class, and color too

Reply:
Also I think you need to add a Description file because the style guide doesn't have CODEOWNERS anymore!